src/EnergyPlus/OutputReportTabular.cc

```diff
@@ -10012,6 +10016,9 @@ namespace OutputReportTabular {
 				if ( Zone( iZone ).FloorArea > 0 && usezoneFloorArea ) {
 					tableBody( 12, iZone ) = RealToStr( totPlugProcess * Wm2_unitConv / Zone( iZone ).FloorArea, 4 );
 				}
+				// Hybrid Model Multiplier Added by Sang Hoon Lee August 2015
+				HMMultiplier = Zone( iZone ).ZoneVolCapMultpSensHMAverage;
```

Comment:
We addressed the issue. Two columns are only shown when the hybrid model internal mass is triggered. One column indicates Y/N of the hybrid model internal mass application for the zone, and the other one for the multiplier value.